content/tutorial/tutorial.md

```diff
@@ -513,63 +519,64 @@ class Board extends React.Component {
 }
 ```
 
-**[View the full code at this point](https://codepen.io/gaearon/pen/ybbQJX?editors=0010)**
+**[अभी तक का पूरा कोड देखें](https://codepen.io/gaearon/pen/ybbQJX?editors=0010)**
 
-After these changes, we're again able to click on the Squares to fill them, the same as we had before. However, now the state is stored in the Board component instead of the individual Square components. When the Board's state changes, the Square components re-render automatically. Keeping the state of all squares in the Board component will allow it to determine the winner in the future.
+इन परिवर्तनों के बाद, यह चंगेस करने के बाद, हम पहले के जैसे Squares को क्लिक कर पा रहे। हालाँकि, अब state को अलग-अलग Square कौम्पोनॅन्ट के बजाय Board कौम्पोनॅन्ट में संग्रहीत किया जाता है। जब Board का state बदल जाता है, तो Square कौम्पोनॅन्ट स्वचालित रूप से फिर से render करते हैं। Board कौम्पोनॅन्ट में सभी स्क्वायरस की स्थिति को ध्यान में रखते हुए यह भविष्य में विजेता का निर्धारण करने की अनुमति देगा।
 
-Since the Square components no longer maintain state, the Square components receive values from the Board component and inform the Board component when they're clicked. In React terms, the Square components are now **controlled components**. The Board has full control over them.
+चूंकि Square कौम्पोनॅन्ट अब state को बनाए नहीं रखते हैं, इसलिए Square कौम्पोनॅन्ट Board कौम्पोनॅन्ट से वैल्यू प्राप्त करते हैं और क्लिक किए जाने पर Board कौम्पोनॅन्ट को सूचित करते हैं। React के संदर्भ में, Sqaure कौम्पोनॅन्ट अब **controlled components** हैं। Board का उन पर पूरा नियंत्रण है।
```

Comment:
इसलिए is unnecessary

Reply:
**controlled components** है। (spelling)

Reply:
> **controlled components** है। (spelling)

I didn't get this? Did you mean spelling is wrong?

Reply:
yes, hain should be hai